src/app/services/data/organisationsStorage.js

```diff
@@ -4,20 +4,44 @@ const Sequelize = require('sequelize');
 
 const Op = Sequelize.Op;
 const logger = require('./../../../infrastructure/logger');
-const { organisations } = require('./../../../infrastructure/repository');
+const { organisations, organisationCategory, organisationStatus, establishmentTypes } = require('./../../../infrastructure/repository');
 
 
-const list = async () => {
+const list = async (includeAssociations = false) => {
```

Comment:
does it need a default of false?

Reply:
Doesn't need it, but doesn't hurt to have it for clarity I feel.